pkg/driver/camera/camera_linux.go

```diff
@@ -303,6 +304,10 @@ func (c *camera) VideoRecord(p prop.Media) (video.Reader, error) {
 }
 
 func (c *camera) Properties() []prop.Media {
+	if c.cam == nil {
```

Comment:
why would the camera be nil?

Reply:
so `c.cam` is set in the `Open` method but devices are registered in the `discover` function. That leaves a window when you can call `Properties` before you call `Open` on a device which will crash your program. I've hit this case while testing.